pkg/messages/messages_en.go

```diff
@@ -311,6 +311,18 @@ var messagesEn = map[string]Message{
 		Type:        codes.FailedPrecondition,
 		Action:      "Please check if there is any error in POD describe related with volume attach",
 	},
+	SubnetIDListNotFound: {
+		Code:        SubnetIDListNotFound,
+		Description: "Cluster subnet list 'vpc_subnet_ids' is not defined",
+		Type:        codes.FailedPrecondition,
+		Action:      "Please check if there exists configmap 'ibm-cloud-provider-data' and the property 'vpc_subnet_ids' contains any subnet entries. Please check 'kubectl get configmap ibm-cloud-provider-data -n kube-system -o yaml'",
```

Comment:
Can you rephrase it to.
"Please check if configmap 'ibm-cloud-provider-data' exists and if the property 'vpc_subnet_ids' contains any subnet entries. Run the command 'kubectl get configmap ibm-cloud-provider-data -n kube-system -o yaml'"